### PR TITLE
fix: Datetime rollover on 00:00

### PR DESCRIPTION
**Pull request by CrageJ**
### Description
Fixed bug where the calculated day wouldn't roll over on 00:00
Added timestamp to logger

### CODE REVIEW

**Review by rayokamoto — APPROVED** [review]

File: src/commands/skullboard.py [hunk 1]
```diff
@@ -2,7 +2,15 @@
 import requests
 import re
 
-from discord import Client, Interaction, AllowedMentions, Embed, app_commands
+from discord import (
```
Comment (phoenixpereira) [inline src/commands/skullboard.py:5]: Remove `Member` and `Message` as they aren't used.

File: src/models/database.py [hunk 2]
```diff
@@ -6,6 +6,8 @@
 
 import aiosqlite
 
+from utils import time
+
```
Comment (phoenixpereira) [inline src/models/database.py:10]: Remove as it seems to be unused.

File: src/utils/time.py [hunk 3]
```diff
@@ -1,8 +1,11 @@
 from datetime import datetime, timezone, timedelta
+import pytz
```
Comment (phoenixpereira) [inline src/utils/time.py:2]: Remove the duplicated imports and remove `timezone` and `timedelta` as they aren't used.

File: src/utils/time.py [hunk 4]
```diff
@@ -15,6 +18,21 @@ def get_current_day():
...
+        timestamp = tz.localize(timestamp)
+    else:
+        timestamp = timestamp.astimezone(tz)
     epoch = datetime(1970, 1, 1, tzinfo=tz)
     days_since_epoch = (timestamp - epoch).days
     return days_since_epoch
+
+
+def get_timestamp_str(timestamp: datetime = datetime.now()):
+    """Generates a string representing the timestamp, in Australian time"""
```
Comment (phoenixpereira) [inline src/utils/time.py:31]: Comment should say ACST or Adelaide time rather than Australian time.

**Review by phoenixpereira — CHANGES_REQUESTED** [review]

**Review by phoenixpereira — APPROVED** [review]

**Pull request merged**